examples/mocks/mocks.go

```diff
@@ -32,26 +35,38 @@ func main() {
 	mock.Add(&mock.Mock{
 		Err: errors.New("Has been an mock error"),
 	})
+
+	// stop mocks server
+	mock.StopTestServer()
+
+	sendMail() // Send Mail
+
+	// start mocks server
+	mock.StartTestServer()
+
 	sendMail() // Response with mock data
 
 	// clear mock
 	mock.Flush()
 	sendMail() // Send Mail
 
+	// stop mocks server
+	mock.StopTestServer()
+
 }
 
 func sendMail() {
 	m := mail.NewV3Mail()
 	content := mail.NewContent("text/html", "<h1>Hello world</h1>This is an example")
 
-	from := mail.NewEmail("envalidMail", "invalidMail@mail.com")
+	from := mail.NewEmail("Nahuel", "ncostamagna@hotmail.com.ar")
```

Comment:
Please revert to the example name and email.

Reply:
Hello @thinkingserious,
the example of how to get started mocks is in line 19 for start and line 42 for end

```go
	mock.StartTestServer() // start mock server
	
	mock.StopTestServer() // end mock server
```

Do I write documentation in a markdown file?

Thanks you! 
Merry christmas and happy new year!